models/models.go

```diff
@@ -222,13 +222,16 @@ func getEngine() (*xorm.Engine, error) {
 	}
 	switch DbCfg.Type {
 	case "mysql":
+		connType := "tcp"
 		if DbCfg.Host[0] == '/' { // looks like a unix socket
-			connStr = fmt.Sprintf("%s:%s@unix(%s)/%s%scharset=utf8&parseTime=true",
-				DbCfg.User, DbCfg.Passwd, DbCfg.Host, DbCfg.Name, Param)
-		} else {
-			connStr = fmt.Sprintf("%s:%s@tcp(%s)/%s%scharset=utf8&parseTime=true",
-				DbCfg.User, DbCfg.Passwd, DbCfg.Host, DbCfg.Name, Param)
+			connType = "unix"
 		}
+		tls := DbCfg.SSLMode
+		if tls == "disable" { // allow (Postgres-inspired) default value to work in MySQL
```

Comment:
What is used to enable SSL encryption on MySQL? If `true`, do we also need to wrap `enabled` to `true`?

Reply:
I chose to consider SSL_MODE a simple pass-through to the driver, whether it's Postgres or Mysql.  Unifying the configuration options of each of the underlying DBMSs is a challenge I'm not prepared to take on (mainly because I don't have the patience and server certs to test all the possibilities).

So, for Postgres, you use the Postgres driver options (listed above in the sample config file), and for Mysql, you use the Mysql driver options.

*However*, just leaving it at that would break anyone who doesn't specify this option in their config file, and just wants the default.  These two lines of code are a special allowance so the app.ini default value ("disable") does the right thing in any circumstance.

Reply:
@JonasFranzDEV What do you think?

Reply:
My opinion is that having disable change to false is ok because then it won't be a breaking change.